about.html

```diff
@@ -0,0 +1,125 @@
+      <div class="aboutme-content">
+        <span class="ham-btn"><img src="./images/mobile_menu.png" alt=""></span>
+        <p class="second-heading">"Hello Art Gallery"</p>
+        <h2 class="aboutme-heading">CREATIVE COMMONS<br>GLOBAL SUMMIT<br>2022</h2>
+        <div class="aboutme-text">
+          <p class="paragraph2">
+            A fun festival where people from more than 80 countries who believe in the value of sharing and openness and create positive change gather and share their thoughts will be held in Korea in October.
+          </p>
+        </div>
+        <p class="contact">Please contact us per Email for any further questions about CC Summit 2022!</p>
+        <p class="email">ccsummit2022@cckorea.org</p>
+      </div>
+      <div class="aboutme-info">
+        <h3 class="aboutme-paragraph">
+          Creative Commons Global Summit 2022 Logo
+        </h3>
+        <hr class="aboutme-line">
+        <p>The logo of CC Global Summit 2022 was decided through the logo competition from 7 march to 12 march.</p>
+        <div class="Image-logo">
+          <img src="./images/logo-first.png " alt="" width="300" height="100">
+        </div>
+      </div>
+      <hr>
+      <div class="Previous-events">
+        <h3>See the past Capablanca's Memorial</h3>
+        <hr class="event-line">
+        <p>Take a look at the last CC Global Summits which took place in Buenos Aires and in Warsaw.</p>
+        <div class="event-cards">
+          <div class="event-card">
+            <img src="./images/event2020.jpg" alt="" width="300" height="200">
+            <p class="year">2019</p>
+            <p class="city">Capablanca's Memorial in Buenos Aires</p>
+          </div>
+          <div class="event-card">
+            <img src="./images/event2021.jpg" alt="" width="300" height="200">
+            <p class="year">2020</p>
+            <p class="city">Capablanca's Memorial in Warsaw</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="partner">
+      <div class="partnerM">
+        <h3>Partners</h3>
+        <div class="p-container">
+          <div class="partner-item1">
+            
+              <img src="./images/mozilla.png" alt="" width="100" height="80">
+              <img src="./images/google6.png" alt="" width="100" height="90">
+              <img src="./images/Firefox.png" alt="" width="70" height="50">
+            </div>
+      
+            <div class="partner-item2">
+              
+                <img src="./images/humb.png" alt="" width="150" height="100">
+                <img src="./images/airbnb.png" alt="" width="100" height="50">
+              
+            </div>
+          
+          </div>
+        </div>
+    </section>
+    <footer class="footer">
+      <div class="last-footer">
+        <img src="./images/cc-logo2.png" alt="">
+        <p class="para-footer">
+          2014 Creative commons Korea.<br>Some Rights Reserved.
+        </p>
+      </div>
+    </footer>
```

Comment:
- [ ] Kindly as asked by the previous review, give the `footer` section in your `About` page `background-color: black;` to make it like the original design.

Your design
![image](https://user-images.githubusercontent.com/35535130/157781481-80db139d-6cb0-40c2-a42d-deee88eba150.png)

Original design
![image](https://user-images.githubusercontent.com/35535130/157781527-503cb544-d66b-4994-bdd8-af66d2a2fbec.png)

Reply:
resolved to a certain extend